mongorestore/mongorestore_test.go

```diff
@@ -61,6 +62,65 @@ func getRestoreWithArgs(additionalArgs ...string) (*MongoRestore, error) {
 	return restore, nil
 }
 
+func TestDeprecatedDBAndCollectionOptions(t *testing.T) {
+	testtype.SkipUnlessTestType(t, testtype.UnitTestType)
+
+	/* As specified in TOOLS-2363, the --db and --collection options
+	are well-defined only for restoration of a single BSON file */
```

Comment:
[nit]
```suggestion
	// As specified in TOOLS-2363, the --db and --collection options
	// are well-defined only for restoration of a single BSON file.
```
[nit] We typically write multi-line comments with the `//` on each line, as opposed to `/* ... */`. This also applies to the comment below.